docs/reference/snapshot-restore/take-snapshot.asciidoc

```diff
@@ -52,7 +52,7 @@ PUT /_snapshot/my_backup/snapshot_2?wait_for_completion=true
 // TEST[skip:cannot complete subsequent snapshot]
 
 The list of indices that should be included into the snapshot can be specified using the `indices` parameter that
```

Comment:
```suggestion
Use the `indices` parameter, which supports <<multi-index,multi-target syntax>>, to specify the list of indices to include in the snapshot.
```

Reply:
As this PR is primary concerned with the multi-index xref, I'm going to leave this as-is for now. It'll create fewer merge conflicts for me in #58513.